_posts/2023-08-23-dashboards-usability-snapshot.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "OpenSearch Dashboards: A snapshot of the current usability"
+authors: 
+  - apasun
+date: 2023-06-08
+meta_keywords: OpenSearch Dashboards, user experience, usability study
+meta_description: Read on to learn more about our usability study with two OpenSearch Dashboards user groups, Producers and Consumers.
+twittercard:
+  description: Learn about our usability study with two OpenSearch Dashboards user groups, Producers and Consumers, and how you can get involved in Dashboards redesign efforts.
+category:
+  - community
+excerpt: 
+  Creating visual representations from large amounts of data aligns with the fundamental mode of human understanding. To monitor data at scale and make strategic decisions, we need tools that can effectively and intuitively visualize data and support sound business decision-making. OpenSearch gives our users visualization capabilities through OpenSearch Dashboards. Dashboards serves as the user interface for OpenSearch features such as security, alerting, index state management, observability, security analytics, and so forth. 
+---
+
+# OpenSearch Dashboards: A snapshot of the current usability 
+
+Creating visual representations from large amounts of data aligns with the fundamental mode of human understanding. To monitor data at scale and make strategic decisions, we need tools that can effectively and intuitively visualize data and support sound business decision-making. OpenSearch gives our users visualization capabilities through OpenSearch Dashboards. Dashboards serves as the user interface for OpenSearch features such as security, alerting, index state management, observability, security analytics, and so forth. 
+
+Since July 2021, when OpenSearch was forked from Kibana 7.10.2, we have been evolving and differentiating OpenSearch Dashboards to meet the users’ demands, especially those that have migrated from Kibana to OpenSearch Dashboards. One such demand is capability to search, view, and visualize data indexed in users’ search solutions. Communicating insights visually often involves visualizing data using bar charts, pie charts, tables, histograms, and maps or using the optimal solution that best aids comprehension of numbers. 
```

Comment:
```suggestion
Since July 2021, when OpenSearch was forked from Kibana 7.10.2, we have been evolving and differentiating OpenSearch Dashboards to meet users’ demands, especially those that have migrated from Kibana to OpenSearch Dashboards. One such demand is the capability to search, view, and visualize data indexed in users’ search solutions. Communicating insights visually often involves visualizing data using bar charts, pie charts, tables, histograms, and maps or using the optimal solution that best aids comprehension of numbers. 
```